### DISCUSSION TITLE
How do i only auto-update certain containers

### DISCUSSION THREAD

**Discussion by gi-yt** [containrrr/watchtower#1603] [Q&A]
Right now, I have watchtower setup to pull and notify but not auto-update using `WATCHTOWER_MONITOR_ONLY=true`. 

However, I would still like to auto-update a few containers.. Is there a simple way to do this?
 
I want to manually update most of my containers and auto-update just a few of them, so https://github.com/containrrr/watchtower/discussions/823 won't apply in my case.

Thanks in advance!

**Comment by gi-yt** [comment]
I tried ` - com.centurylinklabs.watchtower.monitor-only=false` label. I would have to wait to see if it works though :P

**Comment by gi-yt** [comment]
nope that did not work :/

**Reply by piksel** [reply]
You can do it by using two instances of watchtower and scopes, but it's not very convenient :/

**Comment by saugatadatta** [comment]
Running two instances side by side is not working, as the second instances detecting the presence of another, and turning itself off and the delete.

**Reply by piksel** [reply]
Yes, you need to set scopes for them. See https://containrrr.dev/watchtower/running-multiple-instances/

**Comment by gi-yt** [comment]
my solution was to make it autoupdate and add overrides for certain stacks with `com.centurylinklabs.watchtower.monitor-only=true`. Not the best solution out there but it works!
